sealedenumprocessor/src/main/kotlin/com/livefront/sealedenum/internal/WildcardedTypeParameters.kt

```diff
@@ -0,0 +1,20 @@
+package com.livefront.sealedenum.internal
+
+import com.squareup.kotlinpoet.KModifier
+import com.squareup.kotlinpoet.STAR
+import com.squareup.kotlinpoet.TypeSpec
+
+val TypeSpec.wildcardedTypeVariables
```

Comment:
Docs for this one?

Reply:
I added more documentation (and return type) in https://github.com/livefront/sealed-enum/pull/10/commits/5c163ab36be0beebb1b9a685721bbfa92863094a. This was a good place as well to use a KDoc to explain some of the overall strategy.